### PR TITLE
Event log

### PR DESCRIPTION
**Pull request by Danwhy**
## To be merged after #146 

Adds a log to an events table in the database, every time a relevant event (clicking on certain things) happens.

ref #130

### CODE REVIEW

**Comment by Danwhy** [comment]
@rjmk I'll just quickly rebase this, then it's all yours

**Comment by Danwhy** [comment]
@rjmk :+1: Ready

File: lib/hapiLoginConfig.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ const handler = (request, reply) => {
       db.getAllData(email, (err, data) => {
         if (err) return reply(getDataError);
 
+        data.companion_id = rows[0].id;
```
Comment (rjmk) [inline lib/hapiLoginConfig.js]: Any possibility of no rows?

Comment (Danwhy) [inline lib/hapiLoginConfig.js]: That's already being checked a few lines earlier https://github.com/ageuk/companion-app/blob/master/lib/hapiLoginConfig.js#L20

Comment (rjmk) [inline lib/hapiLoginConfig.js]: Nice

**Comment by rjmk** [comment]
Looks good to me!

**Pull request merged**